distribution/lib/Standard/Base/0.0.0-dev/src/Network/HTTP.enso

```diff
@@ -235,66 +204,95 @@ resolve_headers req =
         all_headers + default_content_type
 
 ## PRIVATE
-   Generate body publisher and optional form content boundary
-resolve_body_to_publisher_and_boundary : Request_Body -> Pair BodyPublisher Text
-resolve_body_to_publisher_and_boundary body:Request_Body =
+   Generate body publisher, optional form content boundary and optional hash from the body
+_resolve_body : Request_Body -> Function | Nothing -> Pair BodyPublisher Text
```

Comment:
Maybe a `BodyPublishInfo` type here since it's up to three fields.

Reply:
As @GregoryTravis mentioned - this signature is no longer valid - this method doesn't return a `Pair` anymore.

+1 for adding `Body_Publish_Info` type for return here, will be more readable